city/router.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from dependencies import get_db
```

Comment:
The `get_db` function is used to provide a database session. Ensure that this function is correctly defined in the `dependencies` module, which it is according to the provided file content.